File: 485-max-consecutive-ones/max-consecutive-ones.go
func findMaxConsecutiveOnes(nums []int) int {
    cnt:=0
    max_cnt:=0
    for _,i:= range(nums){
        if(i==1){
            cnt++;
        }else{
            if max_cnt<cnt {
                max_cnt=cnt;
            }
            cnt=0;
        }
    }
    max_cnt=max(max_cnt,cnt);
    return max_cnt;
}

File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max_cnt=0;
        int cnt=0;
        for(int i:nums){
            if(i==1){
                cnt++;
            }else{
                max_cnt=max(cnt,max_cnt);
                cnt=0;
            }
        }
        max_cnt=max(cnt,max_cnt);
        return max_cnt;
    }
};